img/plan.sh

```diff
@@ -42,6 +42,14 @@ do_unpack() {
   return 0
 }
 
+do_prepare() {
+  pushd "${pkg_cache_path}" >/dev/null
+    go mod edit -replace github.com/containerd/containerd@3a3f0aac8819=github.com/containerd/containerd@814b795
+    go get github.com/go-bindata/go-bindata/go-bindata
+  popd >/dev/null
+  return 0
```

Comment:
This should be safe to remove, `build` will fail if any of these return a non-0 exit status so we don't need an explicit return.

Reply:
Removed return code